lib/chunks.js

```diff
@@ -1065,3 +1077,34 @@ module.exports.getTemplateQuestionIdsAsync = async (question) => {
 module.exports.getTemplateQuestionIds = util.callbackify(
   module.exports.getTemplateQuestionIdsAsync
 );
+
+/**
+ * Logs the changes to chunks for a given job.
+ *
+ * @param {{ updatedChunks: ChunkMetadata[], deletedChunks: ChunkMetadata[] }} chunkMetadata
+ * @param {import('./server-jobs').Job} job
+ * @returns
+ */
+module.exports.logChunkChangesToJob = ({ updatedChunks, deletedChunks }, job) => {
+  if (updatedChunks.length === 0 && deletedChunks.length === 0) {
+    return;
+  }
+
+  const lines = [];
+
+  if (updatedChunks.length > 0) {
+    lines.push('Created or updated the following chunks:');
+    updatedChunks.forEach((chunk) => {
+      lines.push(`  ${module.exports.pathForChunk(chunk)}`);
+    });
+  }
```

Comment:
```suggestion
  } else {
    lines.push('No chunks required creation or deletion.');
  }
```

Reply:
I find myself reaching for a word like "updation" here (this branch doesn't cover deletions) - what's the right way to phrase this? Maybe instead of "created or updated" we just say "generated", then we can say "No changes required chunk generation"?

Reply:
I think I landed on something I liked in 7583c53 - how does that look?

Reply:
Looks great!